modules/button/react/lib/Button.tsx

```diff
@@ -34,6 +34,7 @@ export interface ButtonProps
    * Note: not displayed at `small` size
    */
   icon?: CanvasSystemIcon;
+  href?: string;
```

Comment:
Currently, this only exposes `href`, so you can't use things like `target="_blank"`. We could consider `Pick`ing some additional props from 
```tsx
interface AnchorHTMLAttributes {
  download?: any;
  href?: string;
  hrefLang?: string;
  media?: string;
  ping?: string;
  rel?: string;
  target?: string;
  type?: string;
  referrerPolicy?: string;
}
```